app/models/interest.rb

```diff
@@ -1,3 +1,5 @@
 class Interest < ApplicationRecord
-  belongs_to :user
+  self.inheritance_column = :_type_disabled
```

Comment:
the type is reserved for rails and this was shut down type for rails and i can now use this in Interest model